synapse/lib/lmdbslab.py

```diff
@@ -1651,7 +1651,28 @@ def _xact_action(self, calling_func, xact_func, lkey, *args, db=None, **kwargs):
         except lmdb.MapFullError:
             return self._handle_mapfull()
 
-    def putmulti(self, kvpairs, dupdata=False, append=False, db=None):
+    async def putmulti(self, kvpairs, dupdata=False, append=False, db=None):
+
+        # Use a fast path when we have a small amount of data to prevent creating new
+        # list objects when we don't have too.
```

Comment:
```suggestion
        # list objects when we don't have to.
```
Caught it before cisphyx :)